Formula/stellar-xdr.rb

```diff
@@ -1,16 +1,16 @@
 class StellarXdr < Formula
   desc "CLI for encoding and decoding Stellar XDR"
   homepage "https://github.com/stellar/rs-stellar-xdr"
-  url "https://github.com/stellar/rs-stellar-xdr.git", tag: "v20.0.0-rc1"
+  url "https://github.com/stellar/rs-stellar-xdr.git", tag: "v20.0.0"
   head "https://github.com/stellar/rs-stellar-xdr.git", branch: "main"
 
   depends_on "rust" => :build
 
   def install
-    system "cargo", "install", "--features=cli", *std_cargo_args
+    system "cargo", "install", "--locked", "--features=cli", *std_cargo_args
```

Comment:
This should have been locked all along, so that he deps used during installed matched those in the lock file. Fixing it now.

Reply:
Undoing, because turns out `std_cargo_args` does that.